plugins/flytekit-huggingface/setup.py

```diff
@@ -6,7 +6,7 @@
 
 plugin_requires = [
     "flytekit>=1.3.0b2,<2.0.0",
-    "datasets>=2.4.0",
+    "datasets>=2.4.0,<2.19.2",

```

Comment:
nit: I hope they will fix the bug in the next release
```suggestion
    "datasets>=2.4.0,!2.19.2",

```

Reply:
> nit: I hope they will fix the bug in the next release

No problem, thank you